OSDOCS- 18768 [ROSA]: What's New Entry (#111317)
openshift/openshift-docs: modules/rosa-release-notes-Q2-2026.adoc

Users can implement the Modern TLS 1.3 security profile for {product-title} clusters::
+
With this update, you can enhance the security of your managed endpoints by enforcing the Modern Transport Layer Security (TLS) 1.3 security profile. This profile ensures that all communications with the API server and OAuth endpoints comply with the latest security standards.

> Users can configure machine pools for {product-title} clusters::

Reviewer: Users can already configure machine pools, so I'd suggest focusing on what has changed (scaling to zero nodes). Also, you probably don't need the {product-title} here, because machine pools are natively part of a ROSA cluster.

So maybe something like this:

```suggestion
Users can scale machine pools to zero nodes::
```
